artifacts/ArtifactHandling/Import-Artifacts.ps1

```diff
@@ -97,7 +114,7 @@ function Import-Artifacts {
                         }
                     }
 
-                    $IsModifiedBaseApp = $item.Path.IndexOf("sortorder01") -gt -1
```

Comment:
Since this is a default script, I'm not sure if we should hard code your company name in here. What if someone else wants to use this features? I know it was also a magic string before. 

Maybe it is also something we should consider to change in future to have it solved more generic.

Reply:
accepted via Teams